include/grpcpp/support/proto_buffer_reader.h

```diff
@@ -120,6 +122,49 @@ class ProtoBufferReader : public grpc::protobuf::io::ZeroCopyInputStream {
   /// Returns the total number of bytes read since this object was created.
   int64_t ByteCount() const override { return byte_count_ - backup_count_; }
 
+#ifdef GRPC_PROTOBUF_CORD_SUPPORT_ENABLED
+  /// Read the next `count` bytes and append it to the given Cord.
+  virtual bool ReadCord(absl::Cord* cord, int count) {
```

Comment:
From our discussion - Can you please add a comment as to why this is 'virtual' and not 'override'?

Reply:
Done.